content/sensu-go/5.0/api/overview.md

```diff
@@ -8,6 +9,63 @@ menu:
     parent: api
 ---
 
-Sensu Go includes a redesigned, versioned API to make it even easier to manage your monitoring workflows.
+**Sensu Go 5.0 includes API v2.**
 
-_API docs coming soon_
+Sensu Go includes a REST API to make it even easier to manage your monitoring workflows.
+
+### URL format
+
+Sensu API endpoints use the standard URL format `/api/{group}/{version}/namespaces/{namespace}` where:
+
+- `{group}` is the API group. All currently existing Sensu API endpoints are of group `core`.
+- `{version}` is the API version. Sensu Go 5.0 uses API v2.
+- `{namespace}` is the namespace name. The examples in these API docs use the `default` namespace.
+
+### Data format
+
+The API uses JSON formatted requests and responses.
+In terms of [sensuctl output types][1], the Sensu API uses the `json` format, not `wrapped-json`.
+
+### Data size
+
+API responses are limited to 0.512 MB in size.
```

Comment:
This is actually only for the request body size (i.e. the payload when posting to the API).